e2e-tests/README.md

```diff
@@ -25,11 +25,18 @@ rest of Socorro so you're not mixing requirements:
 	$ # your dev virtualenv.
 	$ pip install -r requirements.txt
 
-An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads].
+An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads]. Note that on Linux, you don't need to downgrade your Firefox install in order to use an older version.
```

Comment:
We should correct our docs, GeckoDriver is not alpha and is ready for use. If we come across any issues we should be filing issues against that project.